### ISSUE TITLE
Taskband / Desktop don't resize/reposition when screen layout changes

### ISSUE THREAD

**Issue by rozniak** [rozniak/xfce-winxp-tc#220]
Listing as a bug though it is more a missing feature. The taskband / desktop don't resize/reposition themselves when you change screen res/layout.

This is something I skipped out on for the base implementation, but it does need doing eventually. Need to test against both X11 and Wayland of course, and I'm a lazy bugger so hence it isn't done yet.

**Comment by ChiefFirestartr** [comment]
I'd love it if i can manually resize this, because it is unusable on a 32:9 monitor lol

**Comment by rozniak** [comment]
@ChiefFirestartr Manually resize which part, sorry? :thinking:

**Comment by ChiefFirestartr** [comment]
Oh, just resize the whole thing, kind of like a panel in KDE where you can specify the width. Haha sorry if I wasn't clear. The start button is on one very extreme side of my monitor, and the tray is on the other, and my monitor is 49".

**Comment by synthslvt** [comment]
I would like to add, that being able to change the height more easily without needing to edit the source code would be nice for theming.

**Comment by rozniak** [comment]
Yeah the current thing of hardcoding it to 30px and docked at the bottom isn't final. That is in issue #275 :+1:
